plugins/PagePerformance/tests/UI/PagePerformance_spec.js

```diff
@@ -102,7 +102,7 @@ describe("PagePerformance", function () {
     });
 
     it("performance overlay should work on page titles report", async function () {
-        await page.goto("?" + urlBase + "#?" + generalParams + "&category=General_Actions&subcategory=Actions_SubmenuPageTitles");
+        await page.goto("?" + urlBase + "#?" + generalParams + "&segment=actions>=1&category=General_Actions&subcategory=Actions_SubmenuPageTitles");
```

Comment:
Note: This change does not result in a changed expected file, as the segment covers all visits. But without the other changes, this test would fail